### ISSUE TITLE
Implementation of YOLOv7 in MIT license?

### ISSUE THREAD

**Issue by dsbyprateekg** [WongKinYiu/YOLO#33]
Hi,

Is [https://github.com/WongKinYiu/yolov7](url) also in plan for MIT license implementation?

**Comment by henrytsui000** [comment]
Yes, originally, we were supporting YOLOv7, but we took the services down to improve the code further. We will resume the service soon. Please keep an eye on this project for updates.

**Comment by ksv87** [comment]
what about yolov7 now?
